src/components/molecules/ToDoItem.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Span } from "../atoms/Span";
+import { styled } from "styled-components";
+import { ImageButton } from "../atoms/ImageButton";
+import deleteButton from "../../image/delete.png";
+import backButton from "../../image/restore.png";
+import { useRecoilState } from "recoil";
+import { nowTodoList, solvedTodoList } from "../../recoil/atom.ts";
+function ToDoItem({ todo, isSolved }) {
+  const [nowTodo, setNowTodo] = useRecoilState(nowTodoList);
+  const [solvedTodo, setSolvedTodo] = useRecoilState(solvedTodoList);
+  const deleteButtonClick = () => {
+    const newSolvedList = solvedTodo.filter((item) => item !== todo);
+    setSolvedTodo(newSolvedList);
+  };
+  const backButtonClick = () => {
+    const newSolvedList = solvedTodo.filter((item) => item !== todo);
+    setSolvedTodo(newSolvedList);
+    setNowTodo([...nowTodo, todo]);
+  };
+  const clickNowToDo = (e) => {
+    // SolvedToDo 아이템 클릭했을 경우, 아무 일도 발생하지 않기하기
+    if (isSolved) {
+      return;
+    } else {
+      const targetTodo = e.target.innerText;
+      const newNowTodoList = nowTodo.filter((item) => item !== targetTodo);
+      setNowTodo(newNowTodoList);
+      setSolvedTodo([...solvedTodo, targetTodo]);
+    }
+  };
+  return (
+    <>
+      <StyledItem isSolved={isSolved}>
+        <Span onClick={clickNowToDo}>{todo}</Span>
+
+        {isSolved ? (
+          <ButtonWrapper>
+            <ImageButton src={deleteButton} onClick={deleteButtonClick} />
+            <ImageButton src={backButton} onClick={backButtonClick} />
+          </ButtonWrapper>
+        ) : (
+          ""
+        )}
```

Comment:
이 부분도 
```suggestion
      {isSolved && (
        <ButtonWrapper>
          <ImageButton src={deleteButton} onClick={deleteButtonClick} />
          <ImageButton src={backButton} onClick={backButtonClick} />
        </ButtonWrapper>
      )}
```
isSolved가 아닐때 따로 보여줄 게 없다면 이렇게 처리하는 것도 좋을 것 같습니다!!!